### DISCUSSION TITLE
Selfhosted Umami instance on Docker with external database

### DISCUSSION THREAD

**Discussion by helloworld121** [umami-software/umami#2746] [Q&A]
Hi there,
I am very interested in umami and I want to deploy it as a docker-container in a cluster and configure an external database.
What I read in the documentation is, that docker images are provided, that already contain mysql or postgresql. 
I also read, that the database URL needs to be provided during docker build.

Is there a way to build a docker image without database configuration? And provide the configurations like database url, username and password during deployment/starting by the docker-cluster?

Maybe there is already a documentation which I didn't found?

Thank you :)

**Comment by YIOrainy** [comment]
Disable the image of the DB , and put the URL for the external DB you are using.
